src/build/smart-contracts.md

```diff
@@ -0,0 +1,341 @@
+foo@bar:~$ CUDOS_NODED keys show -a <name> --keyring-backend "$KEYRING"
+
+# For Example, show the address of the account wasm-power
+foo@bar:~$ WASM_POWER_ADDR=$(echo "$KEYRING_PASS" | CUDOS_NODED keys show -a wasm-power --keyring-backend "$KEYRING")
+
+# You can test it with
+echo $WASM_POWER_ADDR
+
+# After running the following command, you must enter your keyring passphrase and press enter (even if the terminal does not notify you to enter the keyring passphrase)
+# gas estimate: 1891864
+foo@bar:~$ RES=$(echo "$KEYRING_PASS" | CUDOS_NODED tx wasm store /usr/cudos/cw20_base.wasm --from wasm-power $TXFLAG -y --keyring-backend "$KEYRING")
+
+# Check the value
+echo $RES
+
+# You can also get the code this way
+foo@bar:~$ CODE_ID=$(echo $RES | jq -r '.logs[0].events[-1].attributes[-1].value'
+echo $CODE_ID
+```
+3. Get balance
+
+**Add some balance in wasm-power, bob and alice using** [faucet](https://explorer.cudos.org/faucet).
+
+4. Instantiate and verify the contract:
+
+Note that under the parameter "name" you can choose any name for your coin. For example, "Cudos coin" or as in our example below we used "DIZZ COIN".
+
+```
+foo@bar:~$ INIT=$(jq -n --arg "wasmpower" $(CUDOS_NODED keys show -a wasm-power --keyring-backend "$KEYRING" --address) '{ "name": "DIZZ COIN", "symbol": "DIZZ", "decimals": 6, "initial_balances": [{ "address": $wasmpower, "amount": "1000000" }], "mint": {"minter": $wasmpower,"cap": "99900000000"}}')
+
+foo@bar:~$ CUDOS_NODED tx wasm instantiate $CODE_ID "$INIT" \
+--from wasm-power --label "CW20" $TXFLAG -y --keyring-backend "$KEYRING"
+
+# check the contract state and account balance
+foo@bar:~$ CUDOS_NODED query wasm list-contract-by-code $CODE_ID $NODE --output json
+
+# fetch contract address
+foo@bar:~$ CONTRACT=$(wasmd query wasm list-contract-by-code $CODE_ID $NODE --output json | jq -r '.contracts[-1]')
+
+echo $CONTRACT
+```
+
+Congrats, you have successfully deployed and instantiated contract.
+
+### Examples
+
+#### Mint CW20 tokens
+
+```
+ foo@bar:~$ MINT=$(jq -n --arg bob $(CUDOS_NODED keys show -a bob --keyring-backend "$KEYRING" --address) '{ "mint": { "recipient": $bob, "amount": "1000000" } }')
+
+{ "mint": { "recipient": "cudos1er4ekhej7xrs8yn45qn5q6485q6dxq0tu33zft", "amount": "1000000" } }
+
+foo@bar:~$ CUDOS_NODED tx wasm execute $CONTRACT "$MINT" \
+    --from wasm-power $TXFLAG -y --keyring-backend "$KEYRING"
+
+foo@bar:~$ BALANCE_OF=$(jq -n --arg bob $(CUDOS_NODED keys show -a bob --keyring-backend "$KEYRING" --address) '{ "balance": { "address": $bob } }')
+foo@bar:~$ echo $BALANCE_OF
+{ "balance": { "address": "cudos1er4ekhej7xrs8yn45qn5q6485q6dxq0tu33zft" } }
+
+foo@bar:~$ CUDOS_NODED query wasm contract-state smart $CONTRACT "$BALANCE_OF" $NODE
+data:
+  balance: "1000000"
+```
+
+#### Transfer CW20 tokens
+
+As a prerequisite, you need to mint tokens that was mentioned in the previous section(Mint CW20 Tokens).
+
+Transfer tokens by running this command:
+
+```
```

Comment:
lines 158-171 are just duplicates of the previous "Mint CW20 tokens" section - lines 135-148 (and therefore also showing incorrect results/balance): should be removed